app/src/main/java/com/infomaniak/swisstransfer/ui/components/transfer/TransferItemList.kt

```diff
@@ -67,97 +66,10 @@ fun TransferItemList(
-            fileName = "Introduction to Turning It Off and On Again (1).pptx",
-            fileSize = 89_723_143L,
-            mimeType = null,
-            localPath = null,
-        ),
-        FileUi(
-            uid = UUID.randomUUID().toString(),
-            fileName = "Learning to Copy and Paste: A Complete Guide (1).docx",
-            fileSize = 237_866_728L,
-            mimeType = null,
-            localPath = null,
-        ),
-        FileUi(
-            uid = UUID.randomUUID().toString(),
-            fileName = "The 5-Step Guide to Not Breaking Your Code (2).txt",
-            fileSize = 57_689_032L,
-            mimeType = null,
-            localPath = null,
-        ),
-        FileUi(
-            uid = UUID.randomUUID().toString(),
-            fileName = "Introduction to Turning It Off and On Again (2).pptx",
-            fileSize = 89_723_143L,
-            mimeType = null,
-            localPath = null,
-        ),
-        FileUi(
-            uid = UUID.randomUUID().toString(),
-            fileName = "Learning to Copy and Paste: A Complete Guide (2).docx",
-            fileSize = 237_866_728L,
-            mimeType = null,
-            localPath = null,
-        ),
-    )
-
-    val transfers = listOf(
-        TransferUi(
-            uuid = UUID.randomUUID().toString(),
-            createdDateTimestamp = Date().time - 30L * 86_400_000L,
-            expirationDateTimestamp = Calendar.getInstance().apply {
-                time = Date()
-                set(Calendar.DATE, get(Calendar.DATE) + 1)
-            }.time.time,
-            sizeUploaded = 57_689_032L,
-            downloadLimit = 10,
-            downloadLeft = 8,
-            message = "Coucou c'est moi le message de description du transfert.",
-            files = files,
-        ),
-        TransferUi(
-            uuid = UUID.randomUUID().toString(),
-            createdDateTimestamp = Date().time - 5L * 86_400_000L,
-            expirationDateTimestamp = Calendar.getInstance().apply {
-                time = Date()
-                set(Calendar.DATE, get(Calendar.DATE) + 4)
-            }.time.time,
-            sizeUploaded = 89_723_143L,
-            downloadLimit = 2,
-            downloadLeft = 2,
-            message = null,
-            files = files,
-        ),
-        TransferUi(
-            uuid = UUID.randomUUID().toString(),
-            createdDateTimestamp = Date().time - 0.5f.toLong() * 86_400_000L,
-            expirationDateTimestamp = Calendar.getInstance().apply {
-                time = Date()
-                set(Calendar.DATE, get(Calendar.DATE) + 7)
-            }.time.time,
-            sizeUploaded = 237_866_728L,
-            downloadLimit = 420_069,
-            downloadLeft = 402_690,
-            message = "3ème transfert. RAS.",
-            files = files,
-        ),
-    )
-
+private fun Preview(@PreviewParameter(TransferUiListPreviewParameter::class) transfers: List<TransferUi>) {
```

Comment:
```suggestion
private fun TransferItemListPreview(@PreviewParameter(TransferUiListPreviewParameter::class) transfers: List<TransferUi>) {
```

Reply:
Setting a customized name for each preview is kinda tedious.
With @FabianDevel, we would prefer to simply name them all `Preview()`.
@LunarX is kinda ok about that.

Reply:
I'd say, if it's the standard way of coding in jetpack compose or the good practice to name them, then maybe we should keep the names. 

But if it's not that recommended then I don't care much either way as I use a live template to automatically generate the preview so that even if we need a name, the name is automatically filled in most of the time

Reply:
After reading the documentation, it looks like there isn't any convention yet.